src/sdk/sdk.ts

```diff
@@ -109,11 +110,17 @@ export class FusionSDK {
 
         const nonce = await this.getNonce(params.walletAddress, params.nonce)
```

Comment:
I don't think we need to use nonce here. No one use batch cancellation
regarding bit invalidation we can use just a random function